README.md

```diff
@@ -370,3 +370,75 @@ column.
 
 Note, when using this expression. the thing you are counting is tags, not events, so if you
 have 10 events, each of which has 10 tags, then a `count()` of `tags_key` will return 100.
+
+
+##  Migrations
+
+Snuba provides functionality to manage ClickHouse database migrations. Migrations
+in Snuba are organized into groups, which typically correspond with features / sets
+of related tables in Snuba. In some cases, groups can provide a mechanism to test
+experimental features in Snuba that should not be rolled out to most users yet.
+Migration groups can be defined as either optional or mandatory. Optional groups
+can be toggled by the user via `settings.SKIPPED_MIGRATION_GROUPS`. In most cases,
+the default in settings.py should not be changed, and doing so may result in
+unexpected behavior.
+
+
+### Commands
+The `snuba migrations` group of commands provides some functions to manage migrations.
+
+#### List migrations
+`snuba migrations list`
+
+- Lists all migrations and their statuses
+
+#### Run all migrations
+`snuba migrations migrate --force`
```

Comment:
What happens if I run this without --force?

Reply:
Added a note about this, it will only work if none of the pending ones are `blocking`